### DISCUSSION TITLE
Comparison

### DISCUSSION THREAD

**Discussion by Asgard1232** [xtclang/xvm#14] [General]
How does Ecstasy compare to Slider ([https://mrsename.blogspot.com/](https://mrsename.blogspot.com/))? In what cases would you use one instead of the other?

**Comment by cpurdy** [comment]
It's pretty hard to even compare these two, since I can't find anything about the language you linked to, other than the two blog entries. It appears to be a Yeti-inspired transpiler to C, with the same fundamental data types as C, but with row types and garbage collection, and a strong purpose in life of being able to call C functions.

So, I'd say that it is very different from the Ecstasy approach, in almost every single way imaginable 🤣 ...

Are you the author? Have you had a chance to look at or use Ecstasy?
